gcloud/streaming/exceptions.py

```diff
@@ -138,7 +133,7 @@ class BadStatusCodeError(HttpError):
     """The request completed but returned a bad status code."""
 
 
-class NotYetImplementedError(GeneratedClientError):
+class NotYetImplementedError(Error):
```

Comment:
Is this class used anywhere? Shouldn't we just use the native type from Python? Also, any method that isn't implemented should be removed (and a bug filed if we intend to do it "one day").